cmd/admin-handlers.go

```diff
@@ -2555,20 +2555,6 @@ func fetchLambdaInfo() []map[string][]madmin.TargetIDStatus {
 		lambdaMap[targetID.Name] = list
 	}
 
-	for _, tgt := range globalEnvTargetList.Targets() {
```

Comment:
Q: are global env declarations for targets deprecated? Should this be documented somewhere else?